src/main/webapp/clusters-view-physical/clusters-view.controller.js

```diff
@@ -5,9 +5,32 @@ angular.module('managementConsole')
     '$scope',
     'modelController',
     'serverGroups',
-    function ($scope, modelController, serverGroups) {
+    'utils',
+    function ($scope, modelController, serverGroups, utils) {
+           $scope.groups = serverGroups;
+           $scope.servers = modelController.getServer().getNodes();
 
-            $scope.groups = serverGroups;
-            $scope.servers = modelController.getServer().getNodes();
+           $scope.clusters = modelController.getServer().getClusters();
+           $scope.gridEvents = [];
 
+            // Loads latest grid events
+           $scope.refreshClusterEvents = function(cluster, maxLines) {
+                cluster.fetchClusterEvents(maxLines).then(
+                function (response) {
+                     angular.forEach(response, function ( event ) {
+                       $scope.gridEvents.push(event);
+                       });
+                  }
+               );
+           }
+
+          $scope.refreshGridEvents = function() {
+                $scope.gridEvents = [];
+                angular.forEach($scope.clusters, function(cluster){
+                  $scope.refreshClusterEvents(cluster, 10);
```

Comment:
@pzapataf make 10 a variable somewhere so we can extract/configure it elsewhere if needed